client/game/board/Board.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+	import Card from "./Card.svelte";
+	import { createEventDispatcher } from "svelte";
+
+	const dispatch = createEventDispatcher();
+
+	let selection = [];
+
+	function selectionChange(e) {
+		if (e.detail.selected) {
+			selection = [...selection, e.detail.card];
+			if (selection.length == 3) {
+				dispatch("play", {
+					selection
+				});
+			}
+		} else {
+			selection = selection.filter(card => JSON.stringify(card) !== JSON.stringify(e.detail.card));
```

Comment:
I'm not sure if we should be relying on `JSON.stringify` to serialize both objects identically. It might be more readable and more reliable to do value-based comparison instead. Though I definitely get what you're going for here, it's not a necessary change.

Reply:
Valid. I will change and add a helper function